src/background.js

```diff
@@ -35,6 +35,12 @@ if (env.name !== 'production') {
     app.setPath('userData', userDataPath + ' (' + env.name + ')');
 }
 
+// Add proxy support startup switches
+app.commandLine.appendSwitch('proxy-bypass-list');
+app.commandLine.appendSwitch('proxy-pac-url');
+app.commandLine.appendSwitch('proxy-server');
+app.commandLine.appendSwitch('no-proxy-server');
```

Comment:
From the documentation page:
> Don't use a proxy server and always make direct connections. Overrides any other proxy server flags that are passed.

So, why would you want to append this if you're wanting to enable proxies? Just curious.

Also, the `app.commandLine.appendSwitch` takes a value on some of these switches you're using which leads me to believe there needs to be a configuration/settings area on just the Electron application for these values.

Or am I completely misunderstanding this whole pull request's purpose?